### PR TITLE
Use CREATE OR REPLACE AGGREGATE

### PR DESCRIPTION
**Pull request by davidkohn88**
From PG12 on, CREATE OR REPLACE is supported for aggregates,
therefore, since we have dropped support for PG11, we can avoid
going through the rigamarole of having our aggregates in a separate
file from the functions we define to support them. Nor do we need to
handle aggregates separately from other functions as their creation
is now idempotent.

### CODE REVIEW

**Comment by davidkohn88** [comment]
Note, I did the naming thing before realizing that you can't actually call aggregates with named parameters, they're there for documentation purposes only. Had a whole plan about the named parameters and calling conventions that that really threw a bit of a wrench in. Anyway, I'm happy to remove the parameter names I added if folks would like, or we can keep them for clarity, let me know and I'll revise the PR and the commit message accordingly.

**Review by svenklemm — APPROVED** [review]

**Comment by davidkohn88** [comment]
Per @svenklemm's suggestion I've removed the naming of aggregate params and just moved to using CREATE OR REPLACE in this PR.

File: sql/bookend.sql [hunk 1]
```diff
@@ -36,3 +36,30 @@ CREATE OR REPLACE FUNCTION _timescaledb_internal.bookend_deserializefunc(bytea,
...
+CREATE OR REPLACE AGGREGATE last(anyelement, "any") (
+    SFUNC = _timescaledb_internal.last_sfunc,
+    STYPE = internal,
+    COMBINEFUNC = _timescaledb_internal.last_combinefunc,
+    SERIALFUNC = _timescaledb_internal.bookend_serializefunc,
+    DESERIALFUNC = _timescaledb_internal.bookend_deserializefunc,
+    PARALLEL = SAFE,
+    FINALFUNC = _timescaledb_internal.bookend_finalfunc,
+    FINALFUNC_EXTRA
+);
```
Comment (mkindahl) [inline sql/bookend.sql:65]: Missing newline at end of file.

**Review by mkindahl — APPROVED** [review]
LGTM.

**Comment by davidkohn88** [comment]
Question @mkindahl @svenklemm - should I add a note above the aggregates that even though these are now idempotent, anything that changes the on-disk format of the partials for these functions will be non-backwards compatible for continuous aggregates so that should be avoided?

**Comment by mkindahl** [comment]
> Question @mkindahl @svenklemm - should I add a note above the aggregates that even though these are now idempotent, anything that changes the on-disk format of the partials for these functions will be non-backwards compatible for continuous aggregates so that should be avoided?

Additional comments that help the next developer is always useful. We are not to good at commenting our decisions.

**Comment by davidkohn88** [comment]
@mkindahl -> can you take this over, I'm going on vacation and I'm a bit confused as to what's going on here, it seems like there was an upstream change that eliminated the file list?

**Pull request merged**